sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiFlushAndFinalizeDoFn.java

```diff
@@ -181,6 +193,13 @@ public void process(PipelineOptions pipelineOptions, @Element KV<String, Operati
           },
           // onSuccess
           c -> {
+            BigQuerySinkMetrics.FlushRowsCounter("ok").inc();
```

Comment:
I'd prefer to not have a magic string here

Reply:
Done.